src/PoolManager.sol

```diff
@@ -307,6 +311,17 @@ contract PoolManager is IPoolManager, Fees, NoDelegateCall, ERC6909Claims {
         emit ProtocolFeeUpdated(id, newProtocolFee);
     }
 
+    function collectProtocolFees(address recipient, uint256 id, uint256 amount)
+        external
+        returns (uint256 amountCollected)
+    {
+        if (msg.sender != address(protocolFeeController)) revert InvalidCaller();
```

Comment:
Yea so because we accrue to the fee controller the owner can't claim without approval. Like feels weird to have that now compared to before when it was in a separate mapping

Reply:
Technically I think we could allow the owner to pull fees from the protocolFeeController with no approval but that is odd behavior and unexpected/outside the standard... That also would mean the `owner` could always transfer fees from a bricked protocolFeeController to a new one though